odoo_backup_sh/security/security.xml

```diff
@@ -3,4 +3,8 @@
     <record id="base.user_admin" model="res.users">
         <field name="groups_id" eval="[(4, ref('odoo_backup_sh.group_manager'))]"/>
     </record>
+
+    <record model="ir.ui.menu" id="odoo_backup_sh.menu_backup_root">
+        <field name="groups_id" eval="[(6, 0, [ref('odoo_backup_sh.group_manager'), ref('odoo_backup_sh.group_user')])]"/>
```

Comment:
if you add user, you don't need to add manager, because manager should have all rights of user group